src/js/reducers/inspector.js

```diff
@@ -21,7 +21,11 @@ function inspectorReducer(state, action) {
         selected: null
       },
       selected: null,
-      expandedLayers: {}
+      expandedLayers: {},
+      scales: {
+        selected: null,
+        show: false
+      }
```

Comment:
@deathbearbrown Another question -- why model scale inspectors separately here? Since only a mark or scale can be selected at any time, can we not just reuse `selected` at the top-level? The pipelines sidebar has a different selection behaviour, so breaking that out separately makes sense. 

This is probably good enough for now, pending a refactor of how the inspectors state is modeled in the store.